server/views/pages/licence/AP.njk

```diff
@@ -24,9 +24,18 @@
             <p>
                 Date of birth: <span class="bold">{{ licence.dateOfBirth | datetimeToDate }}</span>
             </p>
-            <p>
-                Your licence and sentence expire on <span class="bold">{{ licence.licenceExpiryDate | datetimeToDate }}</span>
-            </p>
+            {% if licence.licenceExpiryDate == licence.sentenceEndDate %}
+                <p>
```

Comment:
Should it be a triple === ?

Reply:
I don't think it makes a difference. They cant be anything except strings